src/Microsoft.Identity.Client/Cache/Items/MsalRefreshTokenCacheItem.cs

```diff
@@ -85,7 +85,10 @@ internal override JObject ToJObject()
         {
             var json = base.ToJObject();
 
-            json[StorageJsonKeys.FamilyId] = FamilyId;
+            if (!string.IsNullOrEmpty(FamilyId))
```

Comment:
If we're going to follow this pattern, we should make a method to take the key and the value and do the nullcheck and only set if valid.  will keep these easily to one line and we can call that method on any of the optional fields.

Reply:
Yes, will do. I was looking if it's possible to do it via newtonsoft configuration. I don't think it's possibble because we're at such a low level down the stack.